docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md

```diff
@@ -76,6 +76,8 @@ and process values).
 | prefix                        | This prefix will be appended to every index created by the exporter; must not contain `_` (underscore).                                                                                                                                                                                                                                                           | zeebe-record    |
 | createTemplate                | If `true` missing indexes will be created automatically.                                                                                                                                                                                                                                                                                                          | `true`          |
 | indexSuffixDatePattern        | This suffix will be appended to every index created by the exporter; The pattern is based on the Java [DateTimeFormater](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/time/format/DateTimeFormatter.html) and supports the same syntax. This is useful when indexes should be created in a different interval, like hourly instead of daily. | `"yyyy-MM-dd'"` |
+| numberOfShards                | The number of [shards](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#_static_index_settings) used for each new record index created.                                                                                                                                                                                         | 3               |
+| numberOfReplicas              | The number of shard [replicas](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#dynamic-index-settings) used for each new record index created.                                                                                                                                                                                 | 0               |
```

Comment:
this is the effective change, on other versions table formatting changed causing the changes to appear more than they actually are